src/erlcloud_ddb2.erl

```diff
@@ -1858,32 +2042,46 @@ update_item(Table, Key, Updates, Opts, Config) ->
 
 -type update_table_return() :: ddb_return(#ddb2_update_table{}, #ddb2_table_description{}).
 
--type global_secondary_index_update() :: {index_name(), pos_integer(), pos_integer()}.
--type global_secondary_index_updates() :: [global_secondary_index_update()].
+-type global_secondary_index_update() :: {index_name(), pos_integer(), pos_integer()} |
+                                         {index_name(), delete} |
```

Comment:
@smilerlee out of curiosity: why order is like  `{index_name(), delete}` instead of more erlangis `{delete, index_name()} `?